lambda/src/main/scala/pricemigrationengine/handlers/NotificationHandler.scala

```diff
@@ -113,10 +148,27 @@ object NotificationHandler extends CohortHandler {
         case Newspaper2024           => s"${currencySymbol}${estimatedNewPrice}"
         case GW2024 =>
           s"${currencySymbol}${PriceCap.priceCapForNotification(oldPrice, estimatedNewPrice, GW2024Migration.priceCap)}"
+        case SupporterPlus2024 => s"${currencySymbol}${estimatedNewPrice}"
       }
 
       _ <- logMissingEmailAddress(cohortItem, contact)
 
+      // ----------------------------------------------------
+      // Data for SupporterPlus2024
+      subscription <- Zuora.fetchSubscription(cohortItem.subscriptionName)
```

Comment:
do we only need to call out to zuora if it's SupporterPlus?

Reply:
@tomrf1 
Oh. This is actually a very good point. I should refactor this. I will make it so that a separate function returns the data that is being computed in that section and that function will only call Zuora for SupporterPlus2024. Good suggestion!

Reply:
@tomrf1 
Refactoring done: https://github.com/guardian/price-migration-engine/pull/1056/commits/1349068f26c7748c40c674408914201696386f10